.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "deps/cpp-netlib"]
 	path = deps/cpp-netlib
-	url = https://github.com/anonimal/cpp-netlib.git
-	branch = 0.13-release
+	url = https://github.com/btolfa/cpp-netlib.git
```

Comment:
First, can you please create a branch which points to anonimal/cpp-netlib@7ab74c50a989225ece5f8d6e2ead83a2938afb50 (the current revision of deps/cpp-netlib) so I can send PR to this branch?

Second, my changes consist of 2 parts - modernize of cmake scripts and C++ code changes required to build under MSVC. In fact, I took the changes for MSVC from the master branch of the cpp-netlib.

I don't like the current way of building and linking with cpp-netlib, so I modernized its build scripts too. 

Adding a dependency to cpp-netlib via `add_subdirectory()` makes build scripts more cross-platform. I also have an idea how to build the cryptopp inside kovri's cmake scripts (through [ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html)). Such changes will allow building kovri using only the CMake, without dependency to Makefile (which is difficult to use with MSVC unlike cmake scripts).

Reply:
>First, can you please create a branch which points to anonimal/cpp-netlib@7ab74c5 (the current revision of deps/cpp-netlib) so I can send PR to this branch?

...Why can't you PR to the current revision that's synced upstream? Once that's merged we can bump the submodule.

>In fact, I took the changes for MSVC from the master branch of the cpp-netlib.

That's great, but they should be incorporated into their 0.13-release branch unless otherwise stated.

>Such changes will allow building kovri using only the CMake, without dependency to Makefile

That's great.